plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -418,7 +388,25 @@ func (e *quotaEvaluator) checkRequest(quotas []api.ResourceQuota, a admission.At
 	return quotas, nil
 }
 
-func (e *quotaEvaluator) evaluate(a admission.Attributes) error {
+func (e *quotaEvaluator) Evaluate(a admission.Attributes) error {
+	e.init.Do(func() {
```

Comment:
> just curious, but has anyone else adopted this pattern?

Didn't think to look.  This just let's me keep a clean API and callers don't have to know they need to start the thing once its passed to them.  I think this one is a little different since it doesn't have a cache to prime.